<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCEF - Simple Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QGenericCodeEdit" name="editor"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuPanels">
     <property name="title">
      <string>Panels</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuModes">
     <property name="title">
      <string>Modes</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuStyles">
     <property name="title">
      <string>Styles</string>
     </property>
    </widget>
    <addaction name="menuPanels"/>
    <addaction name="menuModes"/>
    <addaction name="menuStyles"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open" resource="simple_editor.qrc">
     <normaloff>:/example_icons/rc/folder.png</normaloff>:/example_icons/rc/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save" resource="simple_editor.qrc">
     <normaloff>:/example_icons/rc/document-save.png</normaloff>:/example_icons/rc/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPanel">
   <property name="text">
    <string>Panel</string>
   </property>
  </action>
  <action name="actionModes">
   <property name="text">
    <string>Modes</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>d</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCodeEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>pcef.core.editor</header>
  </customwidget>
  <customwidget>
   <class>QGenericCodeEdit</class>
   <extends>QCodeEdit</extends>
   <header>pcef.core</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="simple_editor.qrc"/>
 </resources>
 <connections/>
</ui>
